superset/assets/visualizations/chord.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+import d3 from 'd3';
+import { category21 } from '../javascripts/modules/colors';
+import './chord.css';
+
+function chordViz(slice, json) {
```

Comment:
This function is the heart of this new visualization, the framework will pass it a `slice` and `json` object, and expects you to mutate the dom element referenced in the slice object. It will be called whenever the slice needs to be rendered or re-rendered. Knowing that this function may be called many times, you can use closures here if you want to persists certain things in the module namespace. 

The json objects corresponds to the data returned from running your query and related context. To see exactly what you should expect in here you can click the `json` button in the explore view.

The slice object contains, amongst other things, a reference to the dom element that you should mutate.